### PR TITLE
Fix screen sharing in recent Chrome

### PR DESCRIPTION
**Pull request by dbkr**
Dreadful hack to work around a bug in recent chrome/electron's WebRTC, as explained.

I'm not sure which is the least hideous out of this (ie. repeatedly calling setCodecPreferences and seeing if it crashes each time) or hardcoding the bad codec and skipping it. Opinions welcome. Edit: LiveKit work around this bug by using the receiver codecs (https://github.com/livekit/client-sdk-js/pull/1088) so that's also an option.

This will probably want backporting to the release.

Fixes https://github.com/element-hq/element-desktop/issues/1703

## Checklist

-   [ ] Tests written for new code (and old code if feasible).
-   [ ] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
-   [ ] Linter and other CI checks pass.
-   [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md)).

### CODE REVIEW

File: src/webrtc/call.ts [hunk 1]
```diff
@@ -2381,22 +2381,42 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
...
+                }
             }
         }
 
-        const screenshareVideoTransceiver = this.transceivers.get(
-            getTransceiverKey(SDPStreamMetadataPurpose.Screenshare, "video"),
-        );
-        // setCodecPreferences isn't supported on FF (as of v113)
-        screenshareVideoTransceiver?.setCodecPreferences?.(codecs);
+        //screenshareVideoTransceiver.setCodecPreferences(codecs);
```
Comment (t3chguy) [inline src/webrtc/call.ts:2419]: Commented out code

**Review by t3chguy — APPROVED** [review]
Looks sane otherwise

File: src/webrtc/call.ts (hunk 1 above)
Comment (dbkr) [reply src/webrtc/call.ts]: oops, thanks

**Review by fkwp — APPROVED** [review]
lgtm

**Pull request merged**